README.md

```diff
@@ -4,3 +4,22 @@ A collection of benchmarks related to Polymer Project libraries.
 
 For the runner used by these benchmarks, see
 [tachometer](https://github.com/PolymerLabs/tachometer/)
+
+## Setup
+
+```sh
+git clone git@github.com:PolymerLabs/benchmarks.git
+cd benchmarks/
+npm install
+```
+
+## Shack
+
+Shack is a simplified subset of the
+[Polymer Shop](https://github.com/Polymer/shop) demo app, implemented for
+benchmarking purposes using lit-html, LitElement, and Polymer 3. To compare the
+first-contentful-paint time for each of these implementations, run:
+
+```sh
+npx tach --config shack.json
```

Comment:
Isn't `npx tach` a little dangerous? If `npm i` wasn't run previously, this will first install the `tach` package and then run it, right?

Reply:
Ugh, that's unfortunate, didn't think of that. I wish `npx` defaulted to `npx --no-install` behavior (I actually aliased it to that because I never want to auto-install).

I added a `tach` script, and changed the README to `npm run tach -- --config shack.json`. WDYT? Maybe we should rename the binary back to `tachometer`, and then `npx tachometer` would be safe.

Reply:
Ended up adding `tachometer` as a binary alias, so I think we can safely recommend `npx tachometer` now.